tool/release/version/Bump.kt

```diff
@@ -1,26 +1,17 @@
 package com.vaticle.dependencies.tool.release.version
```

Comment:
We've kept the logic for incrementing the version number, but taken out everything else. The steps are now:
1. increment number in `VERSION` file
2. commit `VERSION` file and push